templates/bake/Controller/controller.twig

```diff
@@ -18,10 +18,9 @@
  */
 #}
 <?php
+declare(strict_types=1);
 namespace {{ namespace }}\Controller{{ prefix }};
 
-use {{ namespace }}\Controller\AppController;
```

Comment:
Isnt this needed if you are in a prefix etc?

Reply:
True, so will have to add this statement conditionally based on whether `prefix` is empty or not.